src/analysis/processing/qgsalgorithmpointstolines.cpp

```diff
@@ -0,0 +1,226 @@
+/***************************************************************************
+                         qgsalgorithmdpointstolines.cpp
+                         ---------------------
+    begin                : November 2020
+    copyright            : (C) 2020 by Stefanos Natsis
+    email                : uclaros at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsalgorithmpointstolines.h"
+#include "qgsvectorlayer.h"
+
+///@cond PRIVATE
+
+QString QgsPointsToLinesAlgorithm::name() const
+{
+  return QStringLiteral( "pointstolines" );
```

Comment:
Can we keep the name as pointstopath, and remove the existing python algorithm please?